hooks/environment

```diff
@@ -1,2 +1,34 @@
-#!/bin/bash
-python "${BASH_SOURCE%/*}/../python/setup-hooks.py"
+#!/usr/bin/env bash
```

Comment:
Maybe add one or two empty lines to aerate the script.

Reply:
"aerate", I love it :)